src/main/java/frc/robot/constants/Constants.java

```diff
@@ -45,4 +48,17 @@ public static RobotMode getRobotMode() {
     if (getRobotType() == RobotType.ROBOT_SIMBOT) return RobotMode.SIM;
     else return RobotBase.isReal() ? RobotMode.REAL : RobotMode.REPLAY;
   }
+
+  public static class Shooter {
```

Comment:
Not sure where ShooterDynamics constants should live. Maybe only put dimensions that don't change per robot here and move everything else into respective classes

Reply:
I'd say we create a Field class inside Constants that contain all info about the field to be accessed across the code, including info like game element placement, dimensions, etc.

Reply:
Yeah, that's what most teams do, FieldConstants